packages/payload/src/fields/config/types.ts

```diff
@@ -413,7 +413,7 @@ export interface FieldBaseClient {
    * Must not be one of reserved field names: ['__v', 'salt', 'hash', 'file']
    * @link https://payloadcms.com/docs/fields/overview#field-names
    */
-  name?: string
+  name: string
```

Comment:
according to the docs this prop is required, so I assume this change is ok, right?

Reply:
Not all fields have names, e.g. Tabs Fields. Thus it's probably better to leave this optional

Reply:
I reverted this, did you finally make it required?

Reply:
Yep, seems like it's working better than I expected. The Tabs field has an `Omit<FieldBaseClient, 'name'>` anyways